rpm/lua-rex.spec

```diff
@@ -0,0 +1,257 @@
+Name:           lrexlib
+Version:        2.9.0
+Release:        1%{?dist}
+Summary:        Regular expression handling library for Lua
+
+Group:          Development/Libraries
+License:        MIT
+URL:            http://lrexlib.luaforge.net/
+Source0:        lrexlib-%{version}.tar.gz
+
+%description
+Lrexlib are bindings of six regular expression library APIs (POSIX, PCRE, PCRE2,
+Oniguruma, Tre and GNU) to Lua.
+
+%package pcre
+Summary: Lua binding of PCRE library
+BuildRequires:  pcre-devel
+BuildRequires:  pkgconfig
+BuildRequires:  tarantool-devel >= 1.9.0.0
+BuildRequires:  luarocks
+Requires:       tarantool >= 1.9.0.0
+Provides:       lrexlib-pcre = %{version}
+
+%description pcre
+Binding of PCRE library
+
+%package pcre2
+Summary: Lua binding of PCRE-2 library
+BuildRequires:  pcre2
```

Comment:
For pcre you require just pcre-devel, but pcre2 and pcre2-devel both here.

Reply:
There is no actual need for pcre2. Removed.